lib/media/closed_caption_parser.js

```diff
@@ -28,10 +30,10 @@ shaka.media.IClosedCaptionParser = class {
    * captions.
    *
    * @param {BufferSource} mediaFragment
-   * @param {function(Array<!shaka.cea.ICaptionDecoder.ClosedCaption>)} onCCData
-   * A callback function to handle the closed captions from parsed data.
+   * @return {Array<!shaka.cea.ICaptionDecoder.ClosedCaption>}
```

Comment:
Should this Array perhaps be non-nullable?  Or is there some reason it might be null?  If so, how would that differ from an empty Array?